pylabrobot/liquid_handling/liquid_handler.py

```diff
@@ -1435,7 +1428,7 @@ def _trigger_callback(self, method_name: str, *args, error: Optional[Exception]
     """
     if callback := self._callbacks.get(method_name):
       callback(*args, error=error, **kwargs)
-    elif error:
+    elif callback is None and error:
```

Comment:
I'd say `elif error is not None` (callback should be established by the if statement above).

Reply:
Agreed, I was attempting to shoehorn in `is None` :p 

> I generally try to explicitly use `is None` in PLR. Perhaps it's a personal quirk, but I like to be consistent :)

I'm still missing the obvious it seems, and am unable to understand what you're asking for here. Please rewrite this to include the `is None` as you see fit.